pytest.ini

```diff
@@ -0,0 +1,8 @@
+# pytest.ini
+[pytest]
+minversion = 6.0
+addopts =  -rE -q --html=./build/reports/tests/test/index.html --junit-xml=./build/test-results/test/TEST-io.vantiq.vantiqsdk.pytest.xml
```

Comment:
the `--html` & `--junit-xml` makes things relatively easily reviewable after build runs, using our local conventions.  Not perfect, but helpful.